test/CMakeLists.txt

```diff
@@ -23,20 +23,32 @@ add_compile_flags("C;CXX"
 string(REPLACE "-Wl,--no-undefined" ""
     CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS}")
 
-if(POLICY CMP0037)
-    if(CMAKE_VERSION VERSION_LESS 3.11)
-        # cmake below 3.11 reserves name test. Use old policy.
-        # https://cmake.org/cmake/help/v3.11/release/3.11.html#other-changes
-        cmake_policy(SET CMP0037 OLD)
-    else()
-        # Starting from cmake 3.11 name test reserved in special
-        # cases and can be used as target name.
-        cmake_policy(SET CMP0037 NEW)
-    endif()
-endif(POLICY CMP0037)
+# luatest and test-run.py use so-called VARDIR, where Unix socket
+# files for testing purposes are placed.
+# By default, VARDIR in luatest and test-run.py set to "/tmp/t",
+# however, we explicitly set it. It allows easily redefine it.
+# Note, GHA workflows set VARDIR explicitly as well.
+set(VARDIR $ENV{VARDIR})
+if (NOT VARDIR)
+  set(VARDIR "/tmp/t")
+endif()
 
 set(UNIT_TEST_TARGETS "")
 
+# If the test output (stdout or stderr) matches these regular
+# expressions the test will fail, regardless of the process exit
+# code, see [1], [2], [3] and [4].
+#
+# 1. https://cmake.org/cmake/help/latest/prop_test/FAIL_REGULAR_EXPRESSION.html
+# 2. https://testanything.org/tap-version-13-specification.html
+# 3. https://github.com/tarantool/tarantool/blob/master/src/lua/tap.lua
+# 4. https://github.com/tarantool/luatest/blob/master/luatest/output/tap.lua
```

Comment:
Nit: A short commit hash or a tag would work better here: this way the link works even if we rename/move/drop the file.

Reply:
Fixed, thanks!

```diff
--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -41,8 +41,8 @@ set(UNIT_TEST_TARGETS "")
 #
 # 1. https://cmake.org/cmake/help/latest/prop_test/FAIL_REGULAR_EXPRESSION.html
 # 2. https://testanything.org/tap-version-13-specification.html
-# 3. https://github.com/tarantool/tarantool/blob/master/src/lua/tap.lua
-# 4. https://github.com/tarantool/luatest/blob/master/luatest/output/tap.lua
+# 3. https://github.com/tarantool/tarantool/blob/a4b4158f2196442308aba7df8f8e21b98b5319fc/src/lua/tap.lua
+# 4. https://github.com/tarantool/luatest/blob/3a78617c2c4a611f6affc9bb5f2120d4e8ea563e/luatest/output/tap.lua
 list(APPEND CTEST_FAIL_REGEXP
   "^not ok"
   "bad plan"
```